djangocms_versioning_filer/templates/admin/filer/folder/directory_listing.html

```diff
@@ -265,4 +268,24 @@ <h2>&nbsp;</h2>
             </form>
         </div>
     </div>
+
+    <!-- Custom dialog modal used for file validation -->
+    <div class="modal micromodal-slide" id="modal-1" aria-hidden="true">
```

Comment:
```suggestion
    <div class="modal micromodal-slide" id="filer-checks-modal" aria-hidden="true">
```

We should make this id a little more unique to the filer project, this package could be reused.